pkg/qemu/qemu.go

```diff
@@ -372,6 +373,17 @@ func Cmdline(cfg Config) (string, []string, error) {
 
 	// We also want to enable vsock and virtfs here, but QEMU does not support vsock and virtfs for macOS hosts
 
+	if y.MountType == "9p" {
+		for i, f := range y.Mounts {
+			expanded, err := localpathutil.Expand(f.Location)
+			if err != nil {
+				return "", nil, err
+			}
+			tag := fmt.Sprintf("mount%d", i)
+			args = append(args, "-virtfs", fmt.Sprintf("local,path=%s,mount_tag=%s,security_model=mapped-xattr", expanded, tag))
```

Comment:
Can we make the security model configurable in the YAML?

Reply:
There is now configuration for VirtFS, as part of the YAML